forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -204,21 +204,33 @@ impl WalletCommands {
                 exact_balance,
                 fixed_unit,
             } => {
+                // if config.chain.name == "calibnet" {
+                //     forest_shim::address::set_current_network(forest_shim::address::Network::Testnet);
+                // }
+
                 let response = wallet_list((), &config.client.rpc_token)
                     .await
                     .map_err(handle_rpc_err)?;
 
                 let default = wallet_default_address((), &config.client.rpc_token)
                     .await
-                    .map_err(handle_rpc_err)?;
+                    .map_err(handle_rpc_err);
 
                 let (title_address, title_default_mark, title_balance) =
                     ("Address", "Default", "Balance");
                 println!("{title_address:41} {title_default_mark:7} {title_balance}");
 
                 for address in response {
                     let addr = address.0.to_string();
-                    let default_address_mark = if addr == default { "X" } else { "" };
+                    let default_address_mark = if let Ok(default) = &default {
```

Comment:
If we don't always have a default address, maybe `wallet_default_address` should return an optional value.

Otherwise, we can write something like `Ok(addr) == default` to simplify the code.

Reply:
`wallet_default_address` is part of the JSON-RPC API that's why I think the result is a string (and string is empty when no default address is found).

Reply:
Reading this code I'm a bit puzzled by usage of a Rust keyword as name of a let binding.

Reply:
`default` is not a keyword, though.

Reply:
That's right it's not in the strict keyword list:
https://doc.rust-lang.org/reference/keywords.html

Reply:
Hmm, I tried `Ok(addr) == default` but didn't managed because `PartialEq` is not implemented for `anyhow::Error`.

Reply:
@elmattic I believe something like this should work.
```
                    let default_address_mark =
                        if default.as_ref().map(|v| v == &addr).unwrap_or(false) {
                            "X"
                        } else {
                            ""
                        };
```